python_modules/jupedsim/jupedsim/models/collision_free_speed.py

```diff
@@ -1,7 +1,10 @@
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
+import warnings
 from dataclasses import dataclass
 
+from deprecated import deprecated
```

Comment:
```suggestion
import sys
if sys.version_info < (3, 13):
    from deprecated import deprecated
else:
    from warnings import deprecated
```

`@deprecated` decorators got added with 3.13, see https://docs.python.org/3/library/warnings.html#warnings.deprecated

Reply:
Apply the same in all places where you imported deprecated